### ISSUE TITLE
Therm doesn't relinquish keyboard focus when switching desktops

### ISSUE THREAD

**Issue by sleach** [trufae/Therm#55]
I have Chrome setup to always be on Desktop 1, and Therm always on Desktop 2. I have Ctrl-1 mapped to switch to Desktop 1, Ctrl-2 to switch to Desktop 2, etc. So when I switch to Desktop 1 using that key combo, Chrome becomes the active app, taking keyboard input etc. 

If I am using Therm on Desktop 2, and then I hit Ctrl-1 to switch to Desktop 1/Chrome, Chrome becomes the "active window" in that the desktop changes, Chrome becomes visible, the System Menu changes to Chrome etc. but Therm is still the target of any keyboard presses. i.e. if I hit Cmd-t for a new tab in Chrome, focus switches back to Therm and a new tab is created there etc. Keyboard focus should be sent to Chrome as soon as I switch away from Therm.

**Comment by trufae** [comment]
This bug is also in iTerm2. Will be good to report it there and backport the patch qhen fixed.

I’ll try to fix some of those mouse-over bugs in therm when i have some spare time(R) and push the fix into iterm later unless they fix that before.

The reason for making therm mouse over only was to make all those bugs more visible for people to care and fix.

Thanks for reporting!

**Comment by sleach** [comment]
Happy to help. Though, I use iTerm2 mainly and I haven't ran into this bug.

**Comment by trufae** [comment]
Did u tried with mouseover option set in iterm?

**Comment by sleach** [comment]
I don't use the mouseover option in iTerm2, I don't use it in Therm either (I verified this problem still happens even with that setting disabled in Therm)

**Comment by trufae** [comment]
You use it in therm. Basically because its hardcoded and the settings are ignored :p
